### PR TITLE
Docs: Add explanation of each component to main readme

### PR DESCRIPTION
**Pull request by stephen304**
Just a first pass at making the documentation more complete. I expect to add more in the component folders including "default" component recommendations and more detailed recommendations under contributing (such as base requirements for new mask/valve/filter designs)

Comments welcome

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,17 +4,50 @@
...
-5. Gaskets / Glue: Every connection point between parts must have an airtight seal with the exception of filter canister lids on the dirty side of the filter media. See the documentation included with each part to ensure the proper seal is achieved. Alternatively, parts that do not need to be disassembled may be glued together with the apropriate glue.
+4. Exhale / Vent Valve: The amount of dead space inside the mask and filters can lead to a risk of asphyxiation. All masks in this repository have provisions for installing an exhale valve. Please be sure to assemble and install one in every mask. Currently, there are no completed valve designs for the masks in this repository. Designs in this repository should not be used in real world scenarios until valves are available.
```
Comment (scottyallen) [inline README.md]: "Currently, there are no completed valve designs for the masks in this repository. Designs in this repository should not be used in real world scenarios until valves are available."

Let's remove this part?

Comment (scottyallen) [reply README.md]: (since it's no longer true?)

File: README.md [hunk 2]
```diff
@@ -4,17 +4,50 @@
...
+5. Gaskets / Glue: Every connection point between parts must have an airtight seal with the exception of filter canister lids on the dirty side of the filter media. See the documentation included with each part to ensure the proper seal is achieved. Alternatively, parts that do not need to be disassembled may be glued together with the appropriate glue.
+
+## Contributing
 
 Pull requests welcome!
```
Comment (scottyallen) [inline README.md]: Can you also add a mention of the discord here, with an invite link?  The link is https://discord.gg/7X2bSzY.

**Comment by scottyallen** [comment]
This is awesome, thank you!  Left just two small comments.

**Review by scottyallen — COMMENTED** [review]
This is awesome, thank you! Left just two small comments.

File: README.md (hunk 2 above)
Comment (stephen304) [reply README.md]: Updated

File: README.md (hunk 1 above)
Comment (stephen304) [reply README.md]: Removed

**Review by scottyallen — APPROVED** [review]
LGTM!

**Pull request merged**